[2025-03-22 13:45:06,514] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:45:19,320] ERROR log_exception() L875  Exception on /vehicule [DELETE] - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py L875 
Traceback (most recent call last):
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py", line 1511, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py", line 919, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\flask_openapi3\scaffold.py", line 117, in view_func
    response = func(**func_kwargs)
  File "D:\Desenvolvimento\PUC\rescue-api\app.py", line 110, in del_produto
    logger.warning(f"Error while delete the vehicule #'{get_vehicule(vehicule_to_update)}', {error_msg}")
                                                        ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "D:\Desenvolvimento\PUC\rescue-api\schemas\vehicule.py", line 19, in get_vehicule
    "id": vehicule.id,
          ^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'id'
[2025-03-22 13:45:19,335] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 13:45:19] "[35m[1mDELETE /vehicule?id=1 HTTP/1.1[0m" 500 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:46:42,543] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:46:42,714] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:47:11,528] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:47:11,685] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:47:14,487] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:47:14,648] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:47:17,174] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:47:17,333] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 13:47:24,772] WARNING del_produto() L110  Error while delete the vehicule #Vehicule not found - call_trace=D:\Desenvolvimento\PUC\rescue-api\app.py L110 
[2025-03-22 13:47:24,775] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 13:47:24] "[33mDELETE /vehicule?id=1 HTTP/1.1[0m" 404 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:23:29,404] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:23:29] "POST /vehicule HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:23:42,634] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:23:42] "POST /vehicule HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:23:56,181] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:23:56] "DELETE /vehicule?id=1 HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:22,459] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:22,614] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:24,343] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:27,907] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:28,066] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:39,309] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:39,469] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:43,574] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:43,723] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:48,317] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:50,803] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:50,966] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:25:56,703] INFO _log() L97   127.0.0.1 - - [22/Mar/2025 14:25:56] "GET /vehicule HTTP/1.1" 200 - - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:26:13,298] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:26:13,447] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:26:33,257] INFO _log() L97    * Detected change in 'D:\\Desenvolvimento\\PUC\\rescue-api\\app.py', reloading - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-03-22 14:26:33,410] INFO _log() L97    * Restarting with stat - call_trace=d:\Desenvolvimento\PUC\rescue-api\.venv\Lib\site-packages\werkzeug\_internal.py L97  
